src/vitest.setup.mjs

```diff
@@ -0,0 +1,15 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```

Comment:
I think this needs to be updated :) https://github.com/testing-library/jest-dom?tab=readme-ov-file#with-vitest

```suggestion
import '@testing-library/jest-dom/vitest';
```

Reply:
it's super confusing, as I understood this should not be necessary when configuring vitest to inject the globals, which I did for compatibility with jest: https://vitest.dev/guide/migration.html#globals-as-a-default

This should install the automatic cleanup and looking at the code it does extend the global expect (if not, tests would fail too).

Looking at the source: https://github.com/testing-library/jest-dom/blob/main/src/vitest.js it's extending the expect from vitest rather than the global (which would have the same effect), but because of the globals option I don't think it's necessary.

Reply:
right, I didn't take into account a possible jest-dom upgrade yet. Our version (v5) doesn't have the `vitest` module yet to import, and I'd like to hold of on upgrading jest-dom for now since it's also used for Storybook tests.